src/Ignore.Tests/GitBasedTests.cs

```diff
@@ -97,14 +111,14 @@ public GitBasedTests(GitRepoFixture gitFixture)
             @"""
 /*.cs
 """,
-            new[] { "foo.cs", "foo/bar/foo.cs", "foo/bar/bar.csproj" });
+            new[] { "foo.cs", "foo/bar/foo.cs", "foo/bar/bar.csproj", "foo_cs" });
 
         [Fact]
         public void SubdirStartsWithStar() => GitBasedTest(
             @"""
 foo/*.cs
 """,
-            new[] { "foo.cs", "foo/bar/foo.cs", "foo/foo.cs", "foo/bar/bar.csproj" });
+            new[] { "foo.cs", "foo/bar/foo.cs", "foo/foo.cs", "foo/bar/bar.csproj", "foo/foo_cs" });
```

Comment:
@goelhardik I did add more tests that highlight the issue that any character was matched. Otherwise it was quite similar.